src/types/importCandidate.types.ts

```diff
@@ -18,9 +18,11 @@ export type ImportCandidateStatus = 'IMPORTED' | 'NOT_IMPORTED' | 'NOT_APPLICABL
 interface ImportStatus {
   candidateStatus: ImportCandidateStatus;
   modifiedDate: string;
+  setBy?: string;
+  nvaPublicationId?: string;
```

Comment:
bare her for status er approved? kan dette bakes inn i typen?

Reply:
Fant en grei implementasjon av det:
```
interface BaseImportStatus {
  candidateStatus: ImportCandidateStatus;
  modifiedDate: string;
}

interface UnimportedImportStatus extends BaseImportStatus {
  candidateStatus: 'NOT_IMPORTED' | 'NOT_APPLICABLE';
}

interface ImportedImportStatus {
  candidateStatus: 'IMPORTED';
  modifiedDate: string;
  setBy: string;
  nvaPublicationId: string;
}

type ImportStatus = UnimportedImportStatus | ImportedImportStatus;
```

men ser ut som data der er litt utforutsigbar, så lar det være som det er med optionals..